content/en/data_jobs/databricks.md

```diff
@@ -87,7 +131,24 @@ Optionally, you can also set other init script parameters and Datadog environmen
 [2]: /getting_started/site/
 
 {{% /tab %}}
-{{% tab "On a specific cluster" %}}
+
+{{% tab "Manually install on a specific cluster" %}}
+
+
+When adding a new Databricks workspace:
```

Comment:
@tiffon remove all of this new next section talking about the integration tile first as part of the manual instructions. I see the logic of why you put it here for completeness assuming the user is starting at the tile, but everything they need to actually do to complete this is in the Databricks console, so we should just tell them to go there from the start and simplify this. I would be annoyed as a user following these steps, and going to the tile, only to not have anything to do/change there.